src/nets/e2e_asr_attctc.py

```diff
@@ -75,7 +75,12 @@ def __call__(self, x):
         self.loss = None
         loss_ctc, loss_att, acc = self.predictor(x)
         alpha = self.mtlalpha
-        self.loss = alpha * loss_ctc + (1 - alpha) * loss_att
+        if alpha == 0:
+            self.loss = loss_att
+        elif alpha == 1:
+            self.loss = loss_ctc
+        else:
+            self.loss = alpha * loss_ctc + (1 - alpha) * loss_att
 
         if self.loss.data < CTC_LOSS_THRESHOLD and not math.isnan(self.loss.data):
```

Comment:
Is it necessary to report ctc or att loss in pure ctc or att mode?

Reply:
This requires a lot of changes including several if-else, and I want to keep the reporter part simple. Also, it is not bad that all modes have the same format.

Reply:
OK.